test/modules/TestResultCoordinator/Settings.cs

```diff
@@ -166,6 +167,7 @@ List<string> GetResultSources()
                 "relayer2.eventHub",
                 "directMethodSender1.send",
                 "directMethodSender2.send",
+                "directMethodSender3.send",
```

Comment:
what are you going to expect if both actual and expected are always the same?

Reply:
I actually have no idea. Dylan is working creating&verify process for DM tests.

Reply:
Ping for EdgeAgent test won't need two stores - it can have only one. I think we should either make a new DirectMethodReport type for this case, or make one of the stores Optional and generate a different report depending on if the option is filled in or not.